integration-test/helpers/pages.js

```diff
@@ -218,7 +239,7 @@ export function loginPage (page, server, opts = {}) {
          */
         async promptWasShown (platform = 'android') {
             const calls = await mockedCalls(page, ['getAutofillData'])
-            expect(calls.length).toBe(1)
+            expect(calls.length).toBeGreaterThan(0)
```

Comment:
This was causing flaky tests because sometimes the suite counted more than one call, but this only happened when parallel tests. What we actually want to know is whether the prompt was shown and that means at least one call.